README.md

```diff
@@ -679,6 +679,61 @@ It also means that these rules will match against following two events :
 
 This behaviour may change in future version (to avoid any confusions) and should not be relied upon.
 
+#### Caveat: JSON parsing and duplicate keys
+
+Events or Rules containing duplicate JSON keys are invalid. When duplicate keys are passed Ruler 
+will only consider the final value in the document order. 
+
+This means following two rules will compile to the same internal representation.
```

Comment:
Minor point: I'd say "…will match the same events" because that's what users care about, not the internal representation. Not a blocker.